tests/test_utils.py

```diff
@@ -101,3 +102,32 @@ def test_flatten_dict(self):
         d = dict(A=1, B=2, C=dict(A=3, B=4, C=dict(A=5, B=6)))
         e = dict(A=1, B=2, C_A=3, C_B=4, C_C_A=5, C_C_B=6)
         self.assertEquals(flatten_dict(d, sep='_'), e)
+
+    def test_parse_env_tags(self):
+        tags = parse_tags_str("key:val")
+        assert tags == dict(key="val")
+
+        tags = parse_tags_str("key:val,key2:val2")
+        assert tags == dict(key="val", key2="val2")
+
+        tags = parse_tags_str("key:val,key2:val2,key3:1234.23")
+        assert tags == dict(key="val", key2="val2", key3="1234.23")
+
+        parsed_tags = parse_tags_str("key:,key3:val1,")
+        assert parsed_tags == dict(key="", key3="val1")
```

Comment:
interesting, do we want empty values?

Reply:
Yeah I played with the idea of disallowing it... Keys I think make definite sense for disallowing, values... maybe?

Reply:
yeah, I don't think values makes sense, it'll just added bytes sent for just the key, and I don't think dogstatsd tags can have an empty value either